### PR TITLE
Update understanding-docker.md

### PR DESCRIPTION
**Pull request by vanpire110**
Modify some wrong words and sentences.

### CODE REVIEW

File: docs/introduction/understanding-docker.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ running code.
 Docker does this by combining a lightweight container virtualization platform
 with workflows and tooling that help you manage and deploy your applications.
 
-At its core, Docker provides a way to run almost any application securely
+At its core, Docker provides a way to run almost any applications securely
```
Comment (thaJeztah) [inline docs/introduction/understanding-docker.md]: actually, I think application (singular) is correct here

File: docs/introduction/understanding-docker.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ Docker images are the **build** component of Docker.
 Docker registries hold images. These are public or private stores from which you
 upload or download images. The public Docker registry is provided with the
 [Docker Hub](http://hub.docker.com). It serves a huge collection of existing
-images for your use. These can be images you create yourself or you can use
+images for your use. These can be images you created for yourself or
```
Comment (thaJeztah) [inline docs/introduction/understanding-docker.md]: this changes the meaning of this sentence; if I create an image myself, it does not mean I only create it for my own use

File: docs/introduction/understanding-docker.md [hunk 3]
```diff
@@ -171,7 +171,7 @@ image. Instructions include actions like:
 * Run a command. 
 * Add a file or directory. 
 * Create an environment variable.
-* What process to run when launching a container from this image.
+* What process to run when launch a container from this image.
```
Comment (thaJeztah) [inline docs/introduction/understanding-docker.md]: launching is correct here

**Comment by thaJeztah** [comment]
Thanks @vanpire110, I added my comments above

File: docs/introduction/understanding-docker.md (hunk 1 above)
Comment (vanpire110) [inline docs/introduction/understanding-docker.md]: Thank you for your opinion. 
If I still want to contribute to this document, should I pull a new request that only modify "AND" to "and", "be" to "is", ", and;" to "." ?

Comment (MHBauer) [inline docs/introduction/understanding-docker.md]: Agree on singular application.

"securely isolated" is what reads weird to me. Maybe "isolated and secure"? Maybe drop `secure` totally. I realize we're trying to play up the security that isolation provides, and docker provides the isolation.

File: docs/introduction/understanding-docker.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ help you in several ways:
 * distributing and shipping those containers to your teams for further development
 and testing
 * deploying those applications to your production environment,
- whether it be in a local data center or the Cloud.
+ whether it is in a local data center or the Cloud.
```
Comment (MHBauer) [inline docs/introduction/understanding-docker.md]: I think either way is fine.

File: docs/introduction/understanding-docker.md (hunk 2 above)
Comment (MHBauer) [inline docs/introduction/understanding-docker.md]: Agree. This does not need to change. 

Aside: I would rewrite that sentence as "These can be images that you create or images that others have created." Although that reads as tautological. Either you made the images or someone else did. What's trying to be said is that a user can upload their own images for reuse, as well as reusing images provided by others.

File: docs/introduction/understanding-docker.md (hunk 3 above)
Comment (MHBauer) [inline docs/introduction/understanding-docker.md]: Agree. This does not need to change.

File: docs/introduction/understanding-docker.md [hunk 5]
```diff
@@ -232,7 +232,7 @@ the file system and a read-write layer is added to the image.
 - **Allocates a network / bridge interface:** Creates a network interface that allows the 
 Docker container to talk to the local host. 
 - **Sets up an IP address:** Finds and attaches an available IP address from a pool. 
-- **Executes a process that you specify:** Runs your application, and; 
+- **Executes a process that you specify:** Runs your application.
```
Comment (MHBauer) [inline docs/introduction/understanding-docker.md]: This looks good. All of the sentences following the bolded parts end in a period. This change would be consistent with that

Aside: the block as a whole  is somewhat inconsistent. The bolded parts are all good in that they continue the sentence of "Docker does the following:". The descriptive sentences that follow each bolded part behave differently, some stand alone, some also seem to complete the sentence "Docker does the following".

**Comment by thaJeztah** [comment]
@vanpire110 you don't need to create a new pull request; if you make the changes, then `git commit --amend`, and do a *force push*, GitHub will automatically update the existing pull request

**Comment by thaJeztah** [comment]
Closing, as this has been replaced by https://github.com/docker/docker/pull/18137

**Pull request closed without merging**